ios/RNTeads.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RNTeads.swift
+//  react-native-teads-sdk-module
+//
+//  Created by Numa Maurin on 21/12/2022.
+//
+
+import Foundation
+import TeadsSDK
+import UIKit
+
+@objc(RNTeads)
+class RNTeads: NSObject {
+    
+    @objc
+    func createInReadPlacement(_ pid : Float, settingsMap : NSDictionary, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) -> Void {
+        print("settingsFromTeads",settingsMap)
+        let data = json(from: settingsMap)!
+        let decoder = JSONDecoder()
+        do {
+            let settings = try decoder.decode(TeadsAdPlacementSettings.self, from: data)
+            let pid = Int(pid)
+            guard let placement = Teads.createInReadPlacement(pid: pid, settings: settings) else {
+                throw NSError()
+            }
+            print("pid placement from sdk",placement.pid)
+        
+            RNTeadsInReadAdInstanceManager.shared.new(placement: placement, pid: placement.pid)
+            resolve(nil)
```

Comment:
even if not really useful from js perspective you should return the placement identifier here
```suggestion
            resolve(pid)
```